docker-compose.yml

```diff
@@ -1,23 +1,52 @@
 version: '2'
 services:
-  backend:
-    image: debian:jessie
-    volumes:
-      - .:/var/www
   nginx:
     build: docker/nginx
+    links:
+      - php
     ports:
-      - 80
+      - ${IP_ADDRESS}:80:80
     volumes_from:
-      - backend
+      - php
   php:
     build: docker/php
-    volumes_from:
-      - backend
+    links:
+      - db:db.${PROJECT}.dev
+      - mailhog
+      - redis
+      - solr
+    volumes:
+      - .:/var/www/html
   db:
     image: mariadb
     environment:
       MYSQL_USER: drupal
       MYSQL_PASSWORD: drupal
       MYSQL_DATABASE: drupal
       MYSQL_ROOT_PASSWORD: root
+    ports:
+      - ${IP_ADDRESS}:3306:3306
+  phpmyadmin:
+    image: phpmyadmin/phpmyadmin:latest
+    environment:
+      UPLOAD_SIZE: 1G
+      PMA_USER: root
+      PMA_PASSWORD: root
+    links:
+      - db
+    ports:
+      - ${IP_ADDRESS}:8080:80
+  mailhog:
+    image: mailhog/mailhog:latest
+    ports:
+      - ${IP_ADDRESS}:8025:8025
+  redis:
+    image: redis
+  solr:
+    image: solr
```

Comment:
I will remove references to Redis and Solr in this branch. I could add them back in in another branch.